pkg/common/table_info_helper.go

```diff
@@ -353,7 +354,7 @@ type columnSchema struct {
 	// ColumnID -> offset in RowChangedEvents.Columns.
 	RowColumnsOffset map[int64]int `json:"row_columns_offset"`
 
-	ColumnsFlag map[int64]*ColumnFlagType `json:"columns_flag"`
```

Comment:
IMHO, the field `ColumnsFlag` is duplicate of the field `RowColFieldTps`, since we can get the flag from the FieldType